docs/user-guide/ingest-data/for-observability/opentelemetry-collector.md

```diff
@@ -0,0 +1,114 @@
+---
+keywords: [OpenTelemetry Collector, OpenTelemetry, traces, data pipeline]
+description: Introduction to using OpenTelemetry Collector to write traces data to GreptimeDB.
+---
+
+# OpenTelemetry Traces
```

Comment:
Is this chapter for traces or collector? 

If it's for traces, we need to mention that collector is an optional component because you can configure greptimedb as otlp destination directly.

If it's for collector, we need to add logs and metrics. Of course we can do it next patch.

Reply:
For now, it's just for traces. I also [mentioned](https://github.com/GreptimeTeam/docs/pull/1619/files#diff-66f086795b635f57c468f011ac6f83dd600cb4818a2f96b2c9c77d79ab96f599R210) that traces data can be written directly to greptimedb.